test/parallel/sdk.API.utils.test.ts

```diff
@@ -190,15 +186,15 @@ test("GIVEN multiSwapBuy WHEN operation is failed AND isMultiSwapAssetTransactio
     ExtrinsicResult.ExtrinsicFailed,
   );
   expect(getEventResultFromMangataTx(multiSwapBuyEvent).data).toEqual(
-    "NoSuchPool",
+    "NotEnoughAssets",
```

Comment:
??

Reply:
Because I changed the error case. We can discuss it.

Reply:
I found another way. We can use BN for a non-existent liquid that will not exist in our tests for a long time. E.g., BN_MILLION.

Reply:
Done